scripts/kwargs-to-map/README.md

```diff
@@ -0,0 +1,68 @@
+You should run this script on a legacy codebase to convert keyword arguments in re-com components to a map.
```

Comment:
Add a massive warning at the top of this README.md as a heading:

WARNING: Do NOT use this script. It has been created in preparation for a future re-com change. No version of re-com (including `master` branch) is yet compatible with the changes made by this script.

And delete "You should run this script... " paragraph.